docs/querying/operators.md

```diff
@@ -205,12 +205,17 @@ or
 
     <aggr-op>([parameter,] <vector expression>) [without|by (<label list>)]
 
-`parameter` is only required for `count_values`, `quantile`, `topk` and
-`bottomk`. `without` removes the listed labels from the result vector, while
+`label_list` is a list of unquoted labels that may include a trailing comma, i.e.
```

Comment:
Above is still label list, without the underscore.

Reply:
Ah, now I get what you tried to tell me in the last review.

Fixed now.